base/server/python/pki/server/cli/acme.py

```diff
@@ -718,8 +734,26 @@ def execute(self, argv):
         database_conf = os.path.join(acme_conf_dir, 'database.conf')
         config = {}
 
-        logger.info('Loading %s', database_conf)
-        pki.util.load_properties(database_conf, config)
+        if database_type:
+            source = '/usr/share/pki/acme/database/%s/database.conf' % database_type
+        else:
+            source = database_conf
+
+        logger.info('Loading %s', source)
+        pki.util.load_properties(source, config)
+
+        # use silent mode if --type or -D was specified
+        if database_type or props:
+
+            logger.info('Setting properties:')
+            for name, value in props.items():
+                logger.info('- %s: %s', name, value)
+                pki.util.set_property(config, name, value)
+
+            pki.util.store_properties(database_conf, config)
```

Comment:
question: Are we trying to load from `source` (in line 743) and store in `database_conf`?

Reply:
See my other comment.